File: ios/AdaptyOnboardingViewManager.m
#import "React/RCTViewManager.h"

@interface RCT_EXTERN_MODULE(AdaptyOnboardingViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(viewId, NSString)
RCT_EXPORT_VIEW_PROPERTY(onboardingJson, NSString)

@end

File: ios/AdaptyOnboardingViewManager.swift
import UIKit
import React

@objc(AdaptyOnboardingViewManager)
public class AdaptyOnboardingViewManager: RCTViewManager {
    public override static func requiresMainQueueSetup() -> Bool {
        return true
    }
    
    public override func view() -> UIView! {
        return AdaptyOnboardingWrapperView()
    }
}
